README.md

```diff
@@ -17,13 +17,16 @@ gem 'uswds-rails', github: '18F/uswds-rails-gem'
 Then bundle and include the stylesheets in your manifest in this order:
 
 ```
+# rename app/assets/stylesheets/application.css to
```

Comment:
Thanks for this PR.

According to http://www.mattboldt.com/organizing-css-and-sass-rails/
perhaps we should be recommending the `@import` statements go into `main.scss` by default? It seems like the `application.css` is the Rails default name and we probably shouldn't mess with that.